src/components/chip/dropdown-chip-multi-select/dropdown-chip-multi-select.test.tsx

```diff
@@ -43,6 +47,38 @@ describe('Dropdown Chip Multi Select', () => {
     expect(onSelectionChange).toBeCalled()
   })
 
+  it('should not handle onSelectionChange if item is disabled', async () => {
+    render(
+      <DropdownChipMultiSelect
+        size={'sm'}
+        onSelectionChange={onSelectionChange}
+        options={mockOptions}
+        label="Sample Label"
+      />,
+    )
+    const label = screen.getByText('Sample Label')
+    await userEvent.click(label)
+    const item = screen.getByText('Sample Item 5')
+    await userEvent.click(item)
+    expect(onSelectionChange).not.toBeCalled()
+  })
+
+  it('should not throw error if onSelectionChange function is not passed', async () => {
+    render(
+      <DropdownChipMultiSelect
+        size={'sm'}
+        onSelectionChange={null as unknown as typeof onSelectionChange}
+        options={mockOptions}
+        label="Sample Label"
+      />,
+    )
+    const label = screen.getByText('Sample Label')
+    await userEvent.click(label)
+    const item = screen.getByText('Sample Item 3')
+    await userEvent.click(item)
+    expect(onSelectionChange).not.toBeCalled()
```

Comment:
@shafin-deriv, `toBeCalled ` is deprecated, I think you should this instead
```suggestion
    expect(onSelectionChange).not.toHaveBeenCalled()
```

Reply:
Done ✅